webpack/ForemanTasks/Components/TasksDashboard/Components/StoppedTasksCard/StoppedTasksCard.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Card } from 'patternfly-react';
+import classNames from 'classnames';
+import { translate as __ } from 'foremanReact/common/I18n';
+import './StoppedTasksCard.scss';
+
+class StoppedTasksCard extends React.Component {
+  getTableItemClassName = (item, type) => {
+    const { focusedOn } = this.props;
+    return focusedOn.normal ||
+      focusedOn.total ||
+      (focusedOn[item] && focusedOn[item][type])
+      ? `${type}-data`
+      : 'not-focused';
+  };
+  shouleBeSelected = focusedOn =>
```

Comment:
What about something like?
```js
['error', 'warning', 'success']
  .map(focuseRowKey => focusedOn[focuseRowKey])
  .map(focusRow => focusRow.total || focusRow.last)
  .includes(true)
```

Reply:
Sadly this doesn't work because there isn't always a value to each of the results for example if `focusedOn = error:{total:true}`
then` focusedOn.warning.total ` will raise` TypeError: Cannot read property 'total' of undefined`

Reply:
See: https://github.com/theforeman/foreman-tasks/pull/395#discussion_r268425445

Reply:
This comment is about the values (the data of the table), this is about is the table cell is focused or not.
If most of the table isn't focused then there will be 'missing' values in the focusedOn object

Reply:
There for, make them `required` (the `focusedOn` values) mean less checks

Reply:
Changed, thanks!